runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftConfig.java

```diff
@@ -32,17 +32,6 @@
 @ConfigGroup("runecraft")
 public interface RunecraftConfig extends Config
 {
-	@ConfigItem(
-		keyName = "showPouch",
-		name = "Show Pouch count",
-		description = "Configures whether the pouch ess count is displayed",
-		position = 1
```

Comment:
Change the position of the other configs

Reply:
I wanted to make the change as small as possible, in case we revert this again in future and implement it in different way